blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -604,10 +698,25 @@ function buildTableRow(campaignJson, createHidden) {
         <div class='property table-column column1'>${campaignJson.name}</div>
         <div class='property table-column column2'>${campaignJson.type}</div>
         <div class='property table-column column3'>${campaignJson.channel}</div>
-        <div class='property table-column column4'>${campaignJson.reviewLinkName}</div>
-        <div class='property table-column column5'>${campaignJson.finalAssetName}</div>
+        <div class='property table-column column4'>
+            <a href="${campaignJson.reviewLinkHref}" class="campaign-link">${campaignJson.reviewLinkName}</a>
+        </div>
+        <div class='property table-column column5'>
+            <a href="${campaignJson.finalAssetHref}" class="campaign-link">${campaignJson.finalAssetName}</a>
```

Comment:
@mdickson-adbe lets hardcode the label to "Final Asset" only link will be provided from CF's